### PR TITLE
K8SPXC-1038 Update for PXC operator release 1.12.0

### PR DESCRIPTION
**Pull request by nmarukovich**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.12.0.pxc-operator.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+          }
+        },
+        "backup": {
+          "8.0.29": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.12.0-pxc8.0-backup",
+            "image_hash": "b737eb09ae878b82d7f654925b466868bf7e914a7f197c3fe3e278cdbc625a1f",
+            "status": "recommended",
+            "critical": false
+          },
+          "2.4.26-1": {
```
Comment (tplavcic) [inline sources/operator.1.12.0.pxc-operator.json:172]: Seems 2.4.26 is official version: https://docs.percona.com/percona-xtrabackup/2.4/release-notes/2.4/2.4.26.html

File: sources/operator.1.12.0.pxc-operator.dep.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "backup": {
+    "8.0.29": {
+      ">=": [
+        {
+          "var": "productVersion"
+        },
+        "8.0"
+      ]
+    },
+    "2.4.26-1": {
```
Comment (tplavcic) [inline sources/operator.1.12.0.pxc-operator.dep.json:11]: Seems 2.4.26 is official version: https://docs.percona.com/percona-xtrabackup/2.4/release-notes/2.4/2.4.26.html

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by hors — APPROVED** [review]

**Pull request merged**